mobile/lib/features/school_management/domain/entities/school.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+
+enum Modality { municipal, estadual, federal, privada }
```

Comment:
In doubt if these categories should be spelled in English

Reply:
yeah, i just copied from backend